core/roboconf-agent/src/main/java/net/roboconf/agent/internal/Agent.java

```diff
@@ -515,4 +517,26 @@ public void setScopedInstance(Instance scopedInstance) {
 	public List<PluginInterface> getPlugins() {
 		return this.plugins;
 	}
+
+
+	/*
+	 * (non-Javadoc)
+	 * @see net.roboconf.agent.AgentMessagingInterface
+	 * #agentStatus()
+	 */
+	@Override
+	public void agentStatus() {
+
+		LinkedBlockingQueue<Message> agentQueue = this.messagingClient.getMessageProcessor().getMessageQueue();
```

Comment:
Where are the unit tests?
IMO, it would be easier for this method to return a string (built with a StringBuilder object). This way, it is easy to verify and we can decide to do whatever we want with the result.